uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/documents.js

```diff
@@ -91,9 +91,9 @@ module.exports = (function () {
     waitForReady: function () {
       var casper = this.casper;
 
-      casper.waitUntilVisible('.ct-container-inner', function () {
-        casper.wait(2000);
-      });
+      casper.waitUntilVisible('.ct-container-inner');
+      casper.waitUntilVisible('.ct-documents-container');
+      casper.waitUntilVisible('.ct-filter-date');
```

Comment:
why the need for 3 different waits?

Reply:
Right, only .ct-filter-date is needed since it is loaded async via the <sidebar-filters> component.